dev_tools/docs/build_api_docs.py

```diff
@@ -15,12 +15,16 @@
 """Tool to generate external api_docs for Cirq (Shameless copy from TFQ)."""
 
 import os
+import types
+from types import FunctionType
 
+import networkx
 from absl import app
 from absl import flags
 from tensorflow_docs.api_generator import doc_controls
 from tensorflow_docs.api_generator import generate_lib
 from tensorflow_docs.api_generator import public_api
+from tensorflow_docs.api_generator.doc_controls import do_not_doc_inheritable
```

Comment:
With this, we have both `import doc_controls` and `from {...}.doc_controls import {...}`. Is the extra import necessary?

Reply:
oh, sure I can clean that up